src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -699,19 +686,8 @@ public function prependExtensionConfig($name, array $config)
      *                                     Set to "true" when you want to use the current ContainerBuilder
      *                                     directly, keep to "false" when the container is dumped instead.
      */
-    public function compile(/*$resolveEnvPlaceholders = false*/)
+    public function compile($resolveEnvPlaceholders = false)
```

Comment:
should also have the `bool` typehint, as it is a new explicit argument in 4.0, as done in other methods